modules/foobar/module.go

```diff
@@ -0,0 +1,22 @@
+package foobar
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/prebid/prebid-server/modules/foobar/config"
+	"github.com/prebid/prebid-server/modules/foobar/hooks"
+)
+
+func Builder(conf json.RawMessage, client *http.Client) (map[string]interface{}, error) {
+	cfg, err := config.New(conf)
+	if err != nil {
+		return nil, err
+	}
+
+	return map[string]interface{}{
+		"code1": hooks.NewValidateHeaderEntrypointHook(cfg),
+		"code2": hooks.NewValidateQueryParamEntrypointHook(cfg),
+		"code3": hooks.NewCheckBodyRawAuctionHook(client, cfg),
+	}, nil
```

Comment:
Yes, the `modules/foobar` is an old one and will be deleted. I left it because I was not sure at that moment if it still needed as my college was working in parallel with that functionality